src/auth/models/strategy.class.ts

```diff
@@ -246,4 +238,12 @@ export abstract class Strategy extends events.EventEmitter {
     public getEvents = (): string[] => {
         return Object.values<string>(AUTH.EVENT)
     }
+
+    public emitIfListenersExist(eventObject: unknown, done: (err: any, id?: unknown) => void, eventName: string) {
```

Comment:
could you put the eventName first, then perhaps use the spread operator to allow for any number of arguments? This way the function could be more generic and applied everywhere

Reply:
ok

Reply:
@paddybasi  done as requested